ci-operator/step-registry/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Run Interop testing..."
+
+echo "This is the start SHARED_DIR: ${SHARED_DIR}"
+echo "ARTIFACT_DIR: ${ARTIFACT_DIR}"
+SECRETS_DIR="/tmp/secrets/ci"
+
+echo "$(ls -all ${SECRETS_DIR})"
+
+unset KUBECONFIG
+
+#/tmp/oc login https://api.fipstest.sivw.p1.openshiftapps.com:6443 --username cluster-admin --password "$(cat $SECRETS_DIR/kubeadmin-password)" || true
+
+#podman info --debug
+
+#podman login -u="$(cat $SECRETS_DIR/username)" -p="$(cat $SECRETS_DIR/password)" quay.io || true
+
+#echo "pulling images..."
+#podman pull quay.io/cspi-qe-images/opp:test || true
+
+#podman pull quay.io/cloudservices/iqe-tests:latest || true
```

Comment:
Instead of using podman here can you try using the oc client to create a pod from the PQE's image on the ephemeral cluster?

Similar to what is being done[ here](https://github.com/openshift/release/pull/39335/files#diff-7f55d09e3509c734d5ea7b7a78422c4f5a1090ff3ee5bb58f54b012e9fe912ccR21)

Reply:
Noted.